src/pages/all-on-one-mini-man/all-on-one-mini-man.html

```diff
@@ -0,0 +1,230 @@
+
+      <ion-col col-4>
+        <div class="form-section">
+          <p>Progress</p>
+          <all-on-one-form-sub-element-hold-no-modal section="progSpeedv1" text="appropriate speed"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="progHesitatev1" text="undue hesitation"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section">
+          <p>Junctions</p>
+          <all-on-one-form-sub-element-hold-no-modal section="juncSpeedv1" text="approach speed"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="juncObsv1" text="observation"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="juncTRightv1" text="turning right"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="juncTLeftv1" text="turning left"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="juncCuttingv1" text="cutting corners"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section">
+          <p>Judgement</p>
+          <all-on-one-form-sub-element-hold-no-modal section="judgeOvertakev1" text="overtaking"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="judgeMeetv1" text="meeting"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="judgeCrossv1" text="crossing"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section bottom-border">
+          <p>Positioning</p>
+          <all-on-one-form-sub-element-hold-no-modal section="posDrivingv1" text="normal driving"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="posLanev1" text="lane discipline"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section bottom-border">
+          <all-on-one-form-sub-element-hold-no-modal section="pedCrossingv1" text="pedestrian crossings"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section bottom-border">
+          <all-on-one-form-sub-element-hold-no-modal section="positionv1" text="position / normal stops"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section bottom-border">
+          <all-on-one-form-sub-element-hold-no-modal section="awarenessv1" text="awareness planning"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+
+        <div class="form-section">
+          <all-on-one-form-sub-element-hold-no-modal section="ancillaryv1" text="ancillary controls"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+      </ion-col>
+    </ion-row>
+
+    <!-- <ion-row>
```

Comment:
Did you mean to commit this commented out markup?

Reply:
not really, just copied over from the all on one page. I'll take it out.